.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
```

Comment:
This is double-quotes versus single-quotes. If you want to take a side, we can enable it and fix the entire code base.